frb_codegen/src/config.rs

```diff
@@ -551,6 +551,22 @@ impl Opts {
         )
         .with_extension("freezed.dart")
     }
+
+    pub fn get_rust_output_paths(&self) -> (PathBuf, PathBuf, PathBuf) {
+        (
+            PathBuf::from(self.rust_output_path.clone()),
+            self.rust_io_output_path(),
+            self.rust_wasm_output_path(),
+        )
+    }
+
+    pub fn get_dart_output_paths(&self) -> (PathBuf, PathBuf, PathBuf) {
+        (
+            PathBuf::from(self.dart_output_path.clone()),
+            self.dart_io_output_path(),
+            self.dart_wasm_output_path(),
+        )
+    }
```

Comment:
uber-nit: would be great to use structs instead of tuples, then it is more readable

For example, a naive idea:

```
struct WhateverName {
  base_path: PathBuf,
  io_path: PathBuf,
  wasm_path: PathBuf,
}
```

Reply:
Refined in bff9223.